modules/snooker/src/main/ts/ephox/snooker/resize/BarManager.ts

```diff
@@ -124,16 +124,21 @@ export const BarManager = (wire: ResizeWire): BarManager => {
   const findClosestEditableTable = (target: SugarElement<Node>): Optional<SugarElement<HTMLTableElement>> =>
     SelectorFind.closest<HTMLTableElement>(target, 'table', isRoot).filter(ContentEditable.isEditable);
 
+  const isResizer = (target: SugarElement<Element>): boolean =>
+    Class.has(target, 'ephox-snooker-resizer-bar') || Class.has(target, 'ephox-dragster-blocker');
+
   /* mouseover on table: When the mouse moves within the CONTENT AREA (NOT THE TABLE), refresh the bars. */
-  const mouseover = DomEvent.bind(wire.view(), 'mouseover', (event) => {
+  const mouseover = DomEvent.bind<MouseEvent, Element>(wire.view(), 'mouseover', (event) => {
     findClosestEditableTable(event.target).fold(
       () => {
         /*
         * mouseout is not reliable within ContentEditable, so for all other mouseover events we clear bars.
         * This is fairly safe to do frequently; it's a single querySelectorAll() on the content and Arr.map on the result.
         * If we _really_ need to optimise it further, we can start caching the bar references in the wire somehow.
+        *
+        * Because the resizers were moved into the editor for inline mode, we need to check if the event target is not a resizer.
         */
-        if (SugarBody.inBody(event.target)) {
+        if (SugarBody.inBody(event.target) && !isResizer(event.target)) {
```

Comment:
Could we add a test for this so hover and try to click on them. On click make sure that the bars are still there or something.